src/community/design-system-day/index.md

```diff
@@ -17,25 +17,27 @@ This means that hyperlinks to /community/design-system-day/ can always encourage
 
 -->
 
-## A series of events throughout 2024
+## Our events
 
 Design System Day is the GOV.UK Design System's flagship event!
 
 It’s an opportunity to collaborate and share knowledge about design systems with like-minded people, covering topics like accessibility, community and decision-making. It's a space for anyone using design systems in government to share ideas and developments, learn from colleagues, and spark conversations on related topics.
 
 You can [sign up to our mailing list](https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system) to get the latest information on Design System Day events planned for this year.
 
+Recordings for our events in 2024 will be available soon.
+
 ## Event 1
 
-The first event this year was hosted online on 21 May 2024. Recordings from the event will be available soon.
+The first event this year was hosted online on 21 May 2024.
```

Comment:
I initially read this as 'this year' as in 'the current year' rather than 'this year' as in 'this year when this event took place (2024)'. Was the latter the intent there?

Reply:
Yeah, I'd just missed this I'll go do and edit to just remove "this year" altogether.